cms/categories/tests.py

```diff
@@ -58,7 +58,7 @@ def test_pages_list(self):
 
         self.assertEqual(table_rows[2].select("td")[0].text, "Post One")
         self.assertEqual(table_rows[2].select("td")[1].text, "News")
-        self.assertEqual(table_rows[2].select("td")[2].text, "04 Feb 2021")
```

Comment:
This doesn't feel right - if we're merging main into the feature, this shouldn't appear as a diff.

I usually prefer rebasing on main rather than merging changes back to the feature.

Reply:
This has resolved itself.